test/lib/parser.test.ts

```diff
@@ -180,30 +180,36 @@ describe("getChordLyricLine", () => {
     assert.deepEqual(result, expected);
   });
 
-  it("should handle only chord line", () => {
-    const result = parser.getChordLyricLine("Em C G D", "");
-    const expected = {
-      className: "line",
-      chordLyricPairs: [
-        {
-          chord: "Em",
-          lyric: " ",
-        },
-        {
-          chord: "C",
-          lyric: " ",
-        },
-        {
-          chord: "G",
-          lyric: " ",
-        },
-        {
-          chord: "D",
-          lyric: "",
-        },
-      ],
-    };
-    assert.deepEqual(result, expected);
+  const emptyLyricsArr = [undefined, "", " "];
```

Comment:
Add regression tests. The tests fail on master and pass on this branch.

<img width="1031" alt="image" src="https://github.com/user-attachments/assets/aeaeb377-4725-47dc-b374-2c1a2fb16fc4" />